lib/taurus/qt/qtgui/graphic/taurusgraphic.py

```diff
@@ -1164,7 +1170,7 @@ def updateStyle(self):
             else:
                 _frName = 'rvalue.magnitude'
             text = self._currText = self.getDisplayValue(fragmentName=_frName)
-        self._currText = text.decode('unicode-escape')
+        self._currText = text
```

Comment:
We need to check if the decode can safely be removed...

Reply:
this line is decoding a str (return of `self.getDisplayValue()`  ) to unicode str
in python3 str does not have decode and the strings are unicode. Should be enough importing str from builtins and doing a cast?